### PR TITLE
Search results use new card template

### PR DESCRIPTION
**Pull request by andygnewman**
- Use new template for card
- Use new template for overall page layout (DMPU on RHS)
- Put ad in 6th slot for smaller screens (as standard stream)

(styling for card summary date will pick up the changes from @bjfletcher changes)

Desktop view
screen shot 2016-06-23 at 13 56 11

Mobile View
screen shot 2016-06-23 at 13 57 17

### CODE REVIEW

File: server/lib/summary-dates.js [hunk 1]
```diff
@@ -9,13 +9,16 @@ module.exports = (items) => {
 		if (!isContent(item)) {
 			return item;
 		}
-		const currItemDate = new Date(item.lastPublished).getDate();
+		const currItemDate = new Date(item.lastPublished);
+		const currItemDateSummary = `${currItemDate.getDate()}.${currItemDate.getMonth()}.${currItemDate.getFullYear()}`;
```
Comment (bjfletcher) [inline server/lib/summary-dates.js]: that or `require('dateformat')(currItemDate, 'dd.mm.yyyy')`

**Comment by bjfletcher** [comment]
@andygnewman looks great to me. I'm a bit :/ that we've got a kind of a hybrid approach to our pages, with some of it using the layout/section configuration, while others hardcoded in HTML templates. Is this the direction we're happy with? Probably one to discuss on Slack. This PR can go in either way. :) (On Monday, that is...)

**Comment by andygnewman** [comment]
Great - have slacked you re: templates

File: server/lib/summary-dates.js (hunk 1 above)
Comment (andygnewman) [inline server/lib/summary-dates.js]: Will take a look at that - saw you'd used that for formatting summary date.

**Pull request merged**